modules/packages/TOML.chpl

```diff
@@ -753,20 +753,28 @@ used to recursively hold tables and respective values
       return false;
     }
 
-    proc set(tbl: string, toml: unmanaged Toml?) {
+    proc set(tbl: string, toml: shared Toml?) {
+      ref t = this(tbl);
+      if t == nil {
+        t = toml;
```

Comment:
I don't understand the purpose of this branch anymore, can we just always assign `t=toml` now? What is the purpose of this overload vs. the one below?